doc/admin/installation/docker_smallscale.rst

```diff
@@ -43,33 +43,33 @@ all lines prepended with a ``$`` symbol can also be run by an unprivileged user.
 Data files
 ----------
 
-First of all, you need a directory on your server that pretix can use to store data files and make that directory
-writable to the user that runs pretix inside the docker container::
+First of all, you need to create a directory on your server that pretix can use to store data files and make that
+directory writable to the user that runs pretix inside the docker container::
 
     # mkdir /var/pretix-data
     # chown -R 15371:15371 /var/pretix-data
 
 Database
 --------
 
-Next, we need a database and a database user. We create this with any kind of database managing tool or directly on
+Next, we need a database and a database user. We can create these with any kind of database managing tool or directly on
 our database's shell, e.g. for MySQL::
 
     $ mysql -u root -p
     mysql> CREATE DATABASE pretix DEFAULT CHARACTER SET utf8 DEFAULT COLLATE utf8_general_ci;
     mysql> GRANT ALL PRIVILEGES ON pretix.* TO pretix@'localhost' IDENTIFIED BY '*********';
     mysql> FLUSH PRIVILEGES;
 
-Replace the asterisks with a password of your own. For MySQL, we will use a unix domain socket to connect to the
-database. For PostgreSQL, be sure to configure the interface binding and your firewall such that the docker container
+Replace the asterisks with a password of your own. For MySQL, we will use a UNIX domain socket to connect to the
```

Comment:
Usually, you use "UNIX" for the actual system "UNIX" and "unix" for the family of UNIX-like systems.